_posts/2018-04-01-upgrade.md

```diff
@@ -4,6 +4,7 @@ title: May 15th Network Upgrade
 subtitle: Bitcoin Cash is adding opcodes and 32mb blocks
 multiLangId: 2018-04-01-upgrade
 lang: en
+permalink: may15hardfork/
```

Comment:
Does not include year. This is worse than the current default url link.

Reply:
While true, it's identical to the currently deployed permalink.  Changing it will break links from news sites, etc, so we can't change it.

Also, this link *is* accessible from the longer link name that contains the rest of the date information.

Reply:
OK, good to know that the longer version with date will still work also.